app/controllers/users.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const User = require('../models').user,
+  logger = require('../logger');
+
+exports.create = (req, res, next) => {
+  const params = req.body.userParams;
```

Comment:
Remember that you can enumerate multiple variables of the same kind using commas; like `const params = req.body.userParams, validationOK = req.body.flagValidationOK`